scrypto-test/src/ledger_simulator/ledger_simulator.rs

```diff
@@ -1238,7 +1238,7 @@ impl<E: NativeVmExtension, D: TestDatabase> LedgerSimulator<E, D> {
     }
 
     pub fn compile<P: AsRef<Path>>(&mut self, package_dir: P) -> (Vec<u8>, PackageDefinition) {
-        Compile::compile(package_dir)
+        Compile::compile(package_dir, CompileProfile::FastWithTraceLogs)
     }
```

Comment:
Can we add a method `compile_with_option`? This is what I expect developers to use if we don't want to the `FastWithTraceLogs` options.